CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Changelog
 
 ### Chores
 
+- Withdrawals applied to the Hardware Wallets transactions ([PR 2408](https://github.com/input-output-hk/daedalus/pull/2408))
```

Comment:
@tomislavhoracek please add this chagne under "Features" and use the following:

> Enabled rewards withdrawals on hardware wallets

Reply:
Done